### ISSUE TITLE
AWS Nginx

### ISSUE THREAD

**Issue by kevmt** [moleculerjs/moleculer#376]
Hi, i have a AWS instance running, and I could run it through pm2 and by `npm run dev` but if i log-in back using ssh is there a way i could come back to the repl of that running moleculer instance? Or do you have a guide somewhere to run moleculerjs in production? I am using only with moleculer-web and created the project with `moleculer init project-simple ...`

**Comment by icebob** [comment]
Yes of course with Moleculer CLI. Use `moleculer connect ...` [Docs](http://moleculer.services/docs/0.12/moleculer-cli.html#Start-a-broker-and-connect-to-a-transporter)

**Comment by kevmt** [comment]
@icebob Thanks!
